### ISSUE TITLE
Postgresql: tags created on the fly are not saved

### ISSUE THREAD

**Issue by waader** [joomla/joomla-cms#6104]
On a postgresql installation: go to the article manager and create a new article or edit an existing article. In the tags field create a tag on the fly or in other words don´t select an existing tag but put in a new tag. Then hit the "Save". The tag field will be cleared. When you pressed "Save & Close" the tag field will be cleared and the article mask will not be closed.

**Comment by alikon** [comment]
i've experienced the same issue  but i'm not sure if my fix is good
the problem is always the same fields declared as NOT NULL  needs a valus

my dirty fix  on line 232 of libraries\cms\helper\tags.php  i've added
```
// Initialize some variables.
		        $db = JFactory::getDbo();
		        $query = $db->getQuery(true);
            $tagTable->checked_out_time = $query->currentTimestamp();
            $tagTable->publish_up = $query->currentTimestamp();
            $tagTable->publish_down = $query->currentTimestamp();
            $tagTable->params = '{}';
            $tagTable->metadesc= ' ';
            $tagTable->metakey= ' ';
            $tagTable->metadata= ' ';
            $tagTable->images= ' ';
            $tagTable->urls= ' ';
```

**Comment by wilsonge** [comment]
This is a very dirty fix ^.^

I think this should be done in the table check rather than in the helper. Urls and images as per other fields should be empty json encoded (like the params is). But :+1: for the principle

**Comment by brianteeman** [comment]
closed as we have a PR #6314 <hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="http://issues.joomla.org/tracker/joomla-cms/6104">issues.joomla.org/joomla-cms/6104</a>.</sub>
